custom/_legacy/pact/utils.py

```diff
@@ -150,6 +153,41 @@ def get_patient_display_cache(case_ids):
     return ret
 
 
+DEFAULT_SIZE = 10
+
+
+def get_base_form_es_query(start=0, size=DEFAULT_SIZE):
+    return (FormES()
+        .remove_default_filters()
+        .domain(PACT_DOMAIN)
+        .filter(filters.term('doc_type', 'XFormInstance'))
+        .start(start)
+        .size(size))
+
+
+def get_base_case_es_query(start=0, size=DEFAULT_SIZE):
+    return (CaseES()
+        .remove_default_filters()
+        .domain(PACT_DOMAIN)
+        .start(start)
+        .size(size))
+
+
+def get_by_case_id_form_es_query(start, size, case_id):
```

Comment:
the order here seems odd, normally you'd put case_id before the pagination options

Reply:
I guess, it's odd. It's just inline with other similar methods such as `get_base_form_es_query` and `get_base_case_es_query`